proposals/lip-0046.md

```diff
@@ -38,9 +38,7 @@ It is worth noting that random seed computation from this commit and reveal proc
 
 ## Specification
 
-In this section, we specify the substores that are part of the Random module store, and the protocol logic called during the block processing and genesis block processing.
-
-The Random module has module ID `MODULE_ID_RANDOM` (see the table below). It is a module with no dependencies.
+In this section, we specify the substores that are part of the Random module store, and the protocol logic called during the block processing and genesis block processing. It is a module with no dependencies.
```

Comment:
I would still say that the Random module has name `MODULE_NAME_RANDOM`.

Reply:
Not sure is necessary, but ok, added